src/main/java/uk/gov/companieshouse/overseasentitiesapi/service/FilingsService.java

```diff
@@ -95,26 +99,120 @@ private void setSubmissionData(Map<String, Object> data, String overseasEntityId
 
         data.put(PRESENTER, submissionDto.getPresenter());
         data.put(ENTITY_FIELD, submissionDto.getEntity());
-        data.put(BENEFICIAL_OWNERS_INDIVIDUAL_FIELD, submissionDto.getBeneficialOwnersIndividual());
+        data.put(BENEFICIAL_OWNERS_INDIVIDUAL_FIELD, getBeneficialOwnersIndividualSubmissionData(submissionDto));
         data.put(BENEFICIAL_OWNERS_GOVERNMENT_OR_PUBLIC_AUTHORITY_FIELD, submissionDto.getBeneficialOwnersGovernmentOrPublicAuthority());
-        data.put(BENEFICIAL_OWNERS_CORPORATE_FIELD, submissionDto.getBeneficialOwnersCorporate());
+        data.put(BENEFICIAL_OWNERS_CORPORATE_FIELD, getBeneficialOwnersCorporateSubmissionData(submissionDto));
         data.put(MANAGING_OFFICERS_INDIVIDUAL_FIELD, submissionDto.getManagingOfficersIndividual());
         data.put(MANAGING_OFFICERS_CORPORATE_FIELD, submissionDto.getManagingOfficersCorporate());
         data.put(BENEFICIAL_OWNERS_STATEMENT, submissionDto.getBeneficialOwnersStatement());
 
-        String trustData = "";
-        if (Objects.nonNull(submissionDto.getTrusts())) {
-            // Convert trust data to JSON string if it exists on transaction else it's to an empty string
-            ObjectMapper mapper = JsonMapper.builder().findAndAddModules().build();
-            try {
-                trustData = mapper.writeValueAsString(submissionDto.getTrusts());
-            } catch (JsonProcessingException e) {
-                throw new ServiceException("Error converting trust data to JSON " + e.getMessage(), e);
+        ApiLogger.debug("Submission data has been set on filing", logMap);
+    }
+
+    private List<BeneficialOwnerIndividualDto> getBeneficialOwnersIndividualSubmissionData(OverseasEntitySubmissionDto submissionDto) throws ServiceException {
+        List<BeneficialOwnerIndividualDto> beneficialOwnersIndividualSubmissionData = new ArrayList<>();
+
+        if (Objects.nonNull(submissionDto.getBeneficialOwnersIndividual())) {
```

Comment:
I would use a guard clause to return early if it is null- 
 if (Objects.isNull(submissionDto.getBeneficialOwnersIndividual())) {
    return beneficialOwnersCorporateSubmissionData;
}
that way you can then remove a layer of 'wrapping'/indentaion from the outer for loop